content/en/agent/guide/network.md

```diff
@@ -27,7 +27,7 @@ further_reading:
 
   - [APM][1] data is `trace.agent.datadoghq.com`
   - [Live Containers][2] data is `process.datadoghq.com`
-  - [Logs][3] data is `agent-intake.logs.datadoghq.com` for TCP traffic
+  - [Logs][3] data is `agent-intake.logs.datadoghq.com` for TCP traffic, `agent-http-intake.logs.datadoghq.eu` in HTTP. Review the list of [logs endpoints][5] for more information.
```

Comment:
Any reason we are using `.eu` here? This and the other URLs could be updated with the site shortcode.

Additionally, the [pipeline](https://gitlab.ddbuild.io/DataDog/documentation/pipelines/3118734) for this branch is failing.